contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -48,8 +50,14 @@ stages:
               python -m pip install pipenv tox
               pipenv install
               artifact="artifact"
-              pipenv run python get_latest_version.py -path $artifact
-              version="$(pipenv run python get_latest_version.py -version)"
+              if [ -z ${VERSION+x} ]
```

Comment:
This intermingles `$()` and `${}`.  Is that intended?

Reply:
This is a good point. Through resolving this I was able to clarify the condition as well.